src/pages/login-page.tsx

```diff
@@ -21,29 +25,44 @@ export const LoginPage = () => {
     setPassword(event.target.value);
   };
 
+  const validate = (email: string, password: string) => {
+    const emailValidation = validateEmail(email);
+    const passwordValidation = validatePassword(password);
+    setEmailError(emailValidation);
+    setPasswordError(passwordValidation);
+    setCredentialsError('');
+    const isValid = emailValidation === '' && passwordValidation === '';
+    return isValid;
```

Comment:
```suggestion
    return emailValidation === '' && passwordValidation === '';
```

Geralmente quando vc cria uma const que é retornada logo em seguida, vc pode retornar direto a expressão.